### ISSUE TITLE
Caught an error when translate dtb to pid

### ISSUE THREAD

**Issue by deliciousdish** [libvmi/libvmi#491]
I caught an error when translate dtb to pid， i turn on DEBUG_ALL and find the error  happened when vmi_translate_kv2p called. I think the root cause of the error is that the vmi->kpgd was destroyed before vmi_translate_kv2p called. vmi->kpgd is set in libvmi/os/linux/core.c by the code following: 
`rc = driver_get_vcpureg(vmi, &vmi->kpgd, CR3, 0);`
Obiviously, vmi->kpgd is set with a randomly selected process's pgd which is exactly running when vmi_init. If the process exits, its pgd will be destroyed and the entry value of vmi->kpgd will be not valid.  I think that's why caught an error when called vmi_translate_kv2p.

**Comment by tklengyel** [comment]
Indeed. It would be better to set kdtb to the dtb of `init_task` or `swapper_pg_dir`.

**Comment by tklengyel** [comment]
Feel free to submit a PR to fix it.
